src/Container/GenericContainer.php

```diff
@@ -12,19 +12,27 @@
 use ReflectionUnionType;
 use function Tempest\attribute;
 use Tempest\Container\Exceptions\CannotAutowireException;
-use Tempest\Container\Exceptions\InvalidInitializerException;
 use Throwable;
 
 final class GenericContainer implements Container
 {
     use HasInstance;
 
     public function __construct(
-        private array $definitions = [],
-        private array $singletons = [],
-        /** @var (Initializer&CanInitialize)[] */
-        private array $initializers = [],
-        private ContainerLog $log = new InMemoryContainerLog(),
+        public array $definitions = [],
```

Comment:
Going back and forth to and from a `ContainerConfig`; I accidentally made these properties public. I'll need to change them back to private, but I'll also need to add methods to get this data, because I need it during testing.

Reply:
Not sure if we need a container config, but I'll reserve the right to change my mind. 😉 

Rather than add a getter for definitions, what about adding a `has` method?